share/settings/remote_test.go

```diff
@@ -81,6 +81,16 @@ func TestRemoteDecode(t *testing.T) {
 			},
 			"localhost:5353:1.1.1.1:53/udp",
 		},
+		{
+			"[::1]:8080:google.com:80",
+			Remote{
+				LocalHost:  "[::1]",
+				LocalPort:  "8080",
+				RemoteHost: "google.com",
```

Comment:
I'm sure it'd work, though could you please add another test with v6 in both local and remote hosts?

Reply:
Done.